tests/UuidTest.php

```diff
@@ -1968,4 +1968,49 @@ public function testUuid5WithZeroName()
 
         $this->assertEquals('b6c54489-38a0-5f50-a60a-fd8d76219cae', $uuid->toString());
     }
+
+    /**
+     * @depends testGetVersionForVersion1
+     */
+    public function testUuidVersionConstantForVersion1()
+    {
+        $uuid = Uuid::fromString('ff6f8cb0-c57d-11e1-9b21-0800200c9a66');
+        $this->assertEquals($uuid->getVersion(), UUID::UUID_TYPE_TIME);
```

Comment:
While it technically passes, please update the class name on the constant here to use the proper capitalization: `Uuid` instead of `UUID`. All the tests below this will also need this change. So, in this example, it should be `Uuid::UUID_TYPE_TIME`. Thanks!